test/support/api/programs.js

```diff
@@ -1,37 +1,55 @@
 import _ from 'underscore';
 import { getResource, getRelationship, mergeJsonApi } from 'helpers/json-api';
 
-import fxPrograms from 'fixtures/test/programs';
+import fxPrograms from 'fixtures/collections/programs';
 
 import { getProgramFlows } from './program-flows';
 import { getProgramActions } from './program-actions';
-import { getWorkspaces } from './workspaces';
+import { workspaceOne, workspaceTwo } from './workspaces';
 
 const TYPE = 'programs';
 
-let programs;
+export const testPrograms = [];
 
-export function getProgram(data, { depth = 0, fixture } = {}) {
+const fxSamplePrograms = _.rest(fxPrograms, 2);
+
+// Exporting only programs needed for testing variance
+export const programOne = getResource(_.extend(fxPrograms[0], {
+  name: 'Program One',
+}), TYPE);
+
+export const programTwo = getResource(_.extend(fxPrograms[1], {
+  name: 'Program Two',
+}), TYPE);
+
+testPrograms.push(programOne, programTwo);
```

Comment:
So this is where the separate runtime issue comes into play? That's why we have to push to the empty `testPrograms` array instead of just doing `const testPrograms = [programOne, programTwo];` here?

Reply:
nope the separate run time is.. if I add `uuid()` on the programOne.. then in the fixtures it would be one uuid but in the tests it would be another.. currently we will get two instances, but that's ok because the data ends up identical.